### PR TITLE
feat: build besu network

### PR DESCRIPTION
**Pull request by Yorkchung**
# PULL REQUEST

## Before 

- [x] 遵守 Commit 規範 (follow [commit convention](https://github.com/cathayddt/bdk/blob/master/.github/COMMIT_CONVENTION.md))
- [x] 遵守 Contributing 規範 (follow [contributing](https://github.com/cathayddt/bdk/blob/master/.github/CONTRIBUTING.md))

## 說明 (Description)

- Add Besu network command support.
- Add network type support to replace Quorum-specific logic.

## 貢獻種類 (Type of change)

- [x] Bug fix (除錯 non-breaking change which fixes an issue)
- [x] New feature (增加新功能 non-breaking change which adds functionality)
- [x] Breaking change (可能導致相容性問題 fix or feature that would cause existing functionality to not work as expected)
- [x] Doc change (需要更新文件 this change requires a documentation update)

**測試環境 (Test Configuration)**:
* OS: 14.2.1
* NodeJS Version: 18.18.2
* NPM Version: 10.5.0
* Docker Version: 24.0.6

## 檢查清單 (Checklist):

- [x] 我的程式碼遵從此專案的規範 (My code follows the style guidelines of this project)
- [x] 我有對於自己的程式碼進行測試檢查 (I have performed a self-review of my own code)
- [x] 我有在程式碼中提供必要的註解 (I have commented my code, particularly in hard-to-understand areas)
- [x] 我有在文件中進行必要的更動 (I have made corresponding changes to the documentation)
- [x] 我的程式碼更動沒有顯著增加錯誤數量 (My changes generate no new warnings)
- [x] 我有新增必要的單元測試 (I have added tests that prove my fix is effective or that my feature works)
- [x] 我有檢查並更正程式碼錯誤的拼字 (I have checked my code and corrected any misspellings)

我已完成以上清單，並且同意遵守 [Code of Conduct](CODE_OF_CONDUCT.md)

I have completed the checklist and agree to abide by the [code of conduct](CODE_OF_CONDUCT.md).

- [x] 同意 (I consent)

### CODE REVIEW

File: src/eth/command/network/add.ts [hunk 1]
```diff
@@ -115,9 +125,9 @@ export const handler = async (argv: Arguments) => {
           ipAddress: ipAddress,
         }
 
-        const spinner = ora('Quorum Network Add ...').start()
+        const spinner = ora(`${networkTypeWithBigFirstLetter} Network Add ...`).start()
         await network.addMemberRemote(addMemberRemoteConfig)
-        spinner.succeed(`Quorum Network Add Member ${memberAddress} Successfully!`)
+        spinner.succeed(`${networkTypeWithBigFirstLetter} Network Add Member ${memberAddress} Successfully!`)
       }
       // TODO: addMemberRemote
```
Comment (kidneyweakx) [inline src/eth/command/network/add.ts:132]: Can delete this TODO (if remote mode use kubernetes)

File: src/eth/command/network/create.ts [hunk 2]
```diff
@@ -37,11 +47,11 @@ export const handler = async (argv: Arguments<OptType>) => {
       const confirmDelete = (await prompts({
         type: 'confirm',
         name: 'value',
-        message: '⚠️ Detecting quorum nodes already exists. The following processes will remove all existing files. Continue?',
+        message: `⚠️ Detecting ${networkType} nodes already exists. The following processes will remove all existing files. Continue?`,
```
Comment (kidneyweakx) [inline src/eth/command/network/create.ts:50]: if I create a network Besu before, but new generate one choose quroum. Wrong output

**Comment by kidneyweakx** [comment]
LGTM!!

A little packages need to fix, run `npm audit --fix` to fix some packages issues

File: src/eth/command/network/create.ts (hunk 2 above)
Comment (kidneyweakx) [reply src/eth/command/network/create.ts:50]: should use fs to read files setting, not networktype

Comment (Yorkchung) [reply src/eth/command/network/create.ts:50]: Fixed this issue and delete origin network when create a new one.

File: src/eth/command/network/add.ts (hunk 1 above)
Comment (Yorkchung) [reply src/eth/command/network/add.ts:132]: removed

File: src/eth/command/network/generate.ts
[the remark this answers is not in the document]
Comment (Yorkchung) [reply src/eth/command/network/generate.ts:1]: added

**Comment by kidneyweakx** [comment]
@johnny30678 You can merged this branch

**Review by johnny30678 — APPROVED** [review]

**Pull request merged**